src/focus_manager/txm_focusable.js

```diff
@@ -54,9 +55,14 @@ export class Focusable {
         if (elmt && elmt.addEventListener) {
             // Add mouse support if possible.
             if (focusManager) {
-                elmt.addEventListener('mouseenter', () => {
+                // Mouse hovering over a focusable item should make it focused, but only if the mouse is
+                // actually moved by the user (as opposed to the view scrolling underneath the mouse).
+                elmt.addEventListener('mouseenter', event => {
                     if (testMouseEnabled && !testMouseEnabled()) return;
-                    focusManager.setFocus(this);
+                    if (focusManager.lastMouseX == event.screenX && focusManager.lastMouseY == event.screenY) return;
```

Comment:
Interesting way to detect lack of movement -- Was expecting something more responsive instead of inferred.  Works well!

Reply:
all other mouse events suffer from the same problem
mouseover would be the candidate, but it still fires when a view scrolls underneath a stationary mouse
so this is pretty much the only way I can think of to do it